zavod/zavod/helpers/sanctions.py

```diff
@@ -30,4 +43,19 @@ def make_sanction(
         sanction.add("country", dataset.publisher.country)
     sanction.add("authority", dataset.publisher.name)
     sanction.add("sourceUrl", dataset.url)
+
+    if program is not None:
+        sanction.set("program", program)
```

Comment:
I'd strongly prefer if we used two separate fields handed into `h.make_sanction`: `program` to be set for the property and `program.key` to be used for the lookup. These /can/ be the same, but forcing them to be will be super complicated and messy. For one thing, it'll auto-activate lookups on all datasets where we use `program`.